tests/ert_tests/ert3/data/test_serializer.py

```diff
@@ -16,14 +13,16 @@
 def test_json_serializer_encode_decode(obj, obj_json):
     json_serializer = _serializer._json_serializer()
 
-    assert json_serializer.encode(obj) == obj_json
+    assert json.loads(json_serializer.encode(obj)) == json.loads(obj_json)
     assert json_serializer.decode(obj_json) == obj
 
     # Test that keyword arguments are passed through
     assert json_serializer.decode(json_serializer.encode(obj, indent=True)) == obj
-    assert json_serializer.encode(obj, separators=(",", ":")) == obj_json.replace(
-        " ", ""
-    )
+
+    # Most compact json serialization:
+    assert json_serializer.encode(
```

Comment:
An alternative method here would be to just drop all whitespaces both from `obj_json` and the result of `json_serializer`. It might be easier to read in the future and it removes the need to ever toggle `indent`.

Reply:
I am not sure I got your question? There are multiple ways to reach a compact json serialization, one way is through the options to `encode()`. This test is basically testing that `separators` is passed on while we are overriding `indent`.

Reply:
Sorry for the late reply. I retract my point :)